core/ansi_escape_parser.js

```diff
@@ -21,10 +21,16 @@ function ANSIEscapeParser(options) {
     events.EventEmitter.call(this);
 
     this.column             = 1;
-    this.row                = 1;
     this.scrollBack         = 0;
     this.graphicRendition   = {};
 
+    if(!_.isNil(options?.startRow)) {
```

Comment:
I'm not sure if you saw it in the diff, but `this.row = Math.min(options.startRow || 1, this.termHeight)` after establishing termHeight ensures we don't end up with wacky values.

...but it should probably check `_.isNumber()` since startRow of 0 is actually acceptable for some wonky terminals.

Reply:
I'm so sorry - I was on mobile when I first looked at your comments, and totally missed this. Should be fixed now. Since we just updated to ES2020 anyway I just used the nullish coalescing operator (??) since that should do the right thing with 0 as well without the extra check. Happy to change it if you want though, lmk.

Reply:
No reason to apologize! I'm happy either way, but we do need to allow 0 and probably should cap at termHeight.